.github/workflows/main.yml

```diff
@@ -52,7 +52,7 @@ jobs:
 
     services:
       postgres:
-        image: postgis/postgis:14-3.4
+        image: postgis/postgis:15-3.3
```

Comment:
What is the latest version of postgres we (arches) support?  IIRC it's 16 for 8 - I know that versions I've deployed in AWS use 16 without issue.  Might be worth upgrading to there.  17 is available too.  https://hub.docker.com/r/postgis/postgis/

Reply:
If we have to make a choice between testing the oldest supported and the newest supported, I would suggest testing older, just to catch pull requests accidentally making use of 16+ functionality, which has happened before.

Reply:
@jacobtylerwalls good point - do we have an official oldest supported version?

Reply:
[I think it's 14](https://github.com/archesproject/arches/blob/4f69be0d1805840d432a61600c44880155df02a6/releases/7.6.0.md#upgrading-an-arches-project), but if we can't get that on the CI runners, then 15 is fine.

Reply:
actually that's fair -- happy to update this PR to not update the Arches postgis image. When it got updated the `main.yml` file in `install/arches-templates` didn't , and v13 was causing build failures